src/base/QXmppStream.cpp

```diff
@@ -47,22 +47,21 @@ static bool randomSeeded = false;
 class QXmppStreamPrivate
 {
 public:
-    QXmppStreamPrivate();
+    QXmppStreamPrivate(QXmppStream *stream);
 
     QString dataBuffer;
     QSslSocket *socket;
 
     // incoming stream state
     QString streamOpenElement;
 
-    bool streamManagementEnabled;
-    QMap<unsigned, QByteArray> unacknowledgedStanzas;
-    unsigned lastOutgoingSequenceNumber;
-    unsigned lastIncomingSequenceNumber;
+    // stream management
+    QXmppStreamManagementManager streamManagementManager;
```

Comment:
I didn't choose `QXmppStreamManager`, because it sounds like it was part of the XMPP Core stream stuff, however `ManagementManager` probably isn't better, so I'll change that. :)

Reply:
Thanks for the review (I'm always happy about better name suggestions :))